### PR TITLE
add new bug_report and feature_request

### PR DESCRIPTION
**Pull request by matex1024**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: input
+    id: contact
+    attributes:
+      label: Contact Details
+      description: How can we get in touch with you if we need more info?
+      placeholder: Please visit out slack chanel [slack](https://adminbro.slack.com/archives/C011WETJSCB) or leave your email.
```
Comment (dziraf) [inline .github/ISSUE_TEMPLATE/bug_report.yml:17]: `out` -> `our`

+ use short link (`https://adminjs.page.link/slack`)

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    validations:
+      required: false
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: Also tell us, what did you expect to happen?
+      placeholder: Tell us what you see!
+      value: "A bug happened!"
```
Comment (dziraf) [inline .github/ISSUE_TEMPLATE/bug_report.yml:26]: Let's write an example bug report here? We don't want people to leave descriptions like `bug happened`

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+      label: Bug prevalence
+      description: "How often do you or others encounter this bug?"
+      placeholder: "Example: Whenever I visit the personal account page (1-2 times a week)"
+    validations:
+      required: true
+  - type: input
+    id: version
+    attributes:
+      label: AdminJS version
+      description: "on which version you encounter this bug?"
```
Comment (dziraf) [inline .github/ISSUE_TEMPLATE/bug_report.yml:41]: I think we should ask for all relevant libraries versions, not just `adminjs`

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+      placeholder: "copy exact version that u are currently using"
```
Comment (dziraf) [inline .github/ISSUE_TEMPLATE/bug_report.yml:42]: Let's not use abbreviations but full words in official communication. `u` -> `you`. Also capitalize the first letter of each sentence/statement for cohesion.

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+    validations:
+      required: true
+  - type: dropdown
+    id: browsers
+    attributes:
+      label: What browsers are you seeing the problem on?
```
Comment (dziraf) [inline .github/ISSUE_TEMPLATE/bug_report.yml:48]: are you seeing -> do you see?

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+  - type: textarea
+    id: logs
+    attributes:
+      label: Relevant log output
+      description: Please copy and paste any relevant log output. This will be automatically formatted into code, so no need for backticks.
+      render: bash
+  - type: textarea
+    id: code
+    attributes:
+      label: Relevant code that giving u issues
```
Comment (dziraf) [inline .github/ISSUE_TEMPLATE/bug_report.yml:64]: that -> that's
u -> you

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+      description: Please copy and paste any relevant code. This will be automatically formatted into code, so no need for backticks.
+      render: TypeScript
```
Comment (dziraf) [inline .github/ISSUE_TEMPLATE/bug_report.yml:66]: Do we have a place for e. g. `Additional info` where people can add screenshots etc?

**Review by dziraf — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 7 above)
Comment (matex1024) [reply .github/ISSUE_TEMPLATE/bug_report.yml:66]: I think you can't attach a file

Comment (matex1024) [reply .github/ISSUE_TEMPLATE/bug_report.yml:66]: sorry, you can but it is just by drag&drop

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: input
+    id: contact
+    attributes:
+      label: Contact Details
+      description: How can we get in touch with you if we need more info?
+      placeholder: Please visit our slack chanel [slack](https://adminjs.page.link/slack) or leave your email.
```
Comment (dziraf) [inline .github/ISSUE_TEMPLATE/bug_report.yml:17]: `chanel` -> `channel`

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 9]
```diff
@@ -0,0 +1,66 @@
...
+      label: Bug prevalence
+      description: "How often do you or others encounter this bug?"
+      placeholder: "Example: Whenever I visit the personal account page (1-2 times a week)"
+    validations:
+      required: true
+  - type: textarea
+    id: versions
+    attributes:
+      label: AdminJS dependencies version
+      description: "Provide to us list of dependencies from package.json "
```
Comment (dziraf) [inline .github/ISSUE_TEMPLATE/bug_report.yml:41]: `Provide to us list of dependencies` -> `Please provide a list of dependencies ...`

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 10]
```diff
@@ -0,0 +1,66 @@
...
+      placeholder: "copy exact versions of plugins that you are currently using"
```
Comment (dziraf) [inline .github/ISSUE_TEMPLATE/bug_report.yml:42]: `copy` -> `Copy`

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 11]
```diff
@@ -0,0 +1,66 @@
...
+    validations:
+      required: true
+  - type: dropdown
+    id: browsers
+    attributes:
+      label: What browsers do you see the problem occured?
```
Comment (dziraf) [inline .github/ISSUE_TEMPLATE/bug_report.yml:48]: `problem occured?` -> `problem on?`

**Review by dziraf — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 11 above)
Comment (matex1024) [reply .github/ISSUE_TEMPLATE/bug_report.yml:48]: "problem on" is not grammatically correct

Comment (dziraf) [reply .github/ISSUE_TEMPLATE/bug_report.yml:48]: `problem on` is grammatically correct, `problem occured` isn't.

**Review by dziraf — APPROVED** [review]

**Pull request merged**